src/main/java/rx/internal/operators/BackpressureUtils.java

```diff
@@ -45,7 +45,9 @@ private BackpressureUtils() {
      * @param n
      *            the number of requests to add to the requested count
      * @return requested value just prior to successful addition
+     * @deprecated Android has issues with reflection-based atomics
```

Comment:
s/Android/Samsung/ 😀 

Let's not lump all of Android in with the likes of the insane OS developers at Samsung who modify Java system packages needlessly without rhyme or reason.